doc/source/cluster/kubernetes/doc_sanitize.cfg

```
@@ -10,14 +10,38 @@ replace: RAYCLUSTER-HEAD-POD-NAME
 regex: raycluster-kuberay-head-[a-z0-9]{5}
 replace: RAYCLUSTER-WORKER-POD-NAME
 
+[rayservice-sample-raycluster-name]
+regex: rayservice-sample-raycluster-[a-z0-9]{5}(?!-)
+replace: RAYSERVICE-SAMPLE-RAYCLUSTER-NAME
+
+[rayservice-sample-head-pod-name]
+regex: rayservice-sample-raycluster-[a-z0-9]{5}-head-[a-z0-9]{5}
+replace: RAYSERVICE-SAMPLE-HEAD-POD-NAME
+
+[rayservice-sample-worker-pod-name]
+regex: rayservice-sample-raycluster-[a-z0-9]{5}-small-group-worker-[a-z0-9]{5}
+replace: RAYSERVICE-SAMPLE-WORKER-POD-NAME
+
+[rayservice-sample-k8s-svc]
+regex: rayservice-sample-raycluster-[a-z0-9]{5}-head-svc
+replace: RAYSERVICE-SAMPLE-K8S-SVC
+
 [rayjob-submit-name]
 regex: raysubmit_[a-zA-Z0-9]{16}
 replace: RAYJOB-SUBMIT-NAME
 
 [time-duration]
-regex: \d+m\d+s|\d+m|\d+s
+regex: (\s)(\d+m\d+s|\d+m|\d+s)(\s)
```

```suggestion
regex: \s\d+m\d+s|\d+m|\d+s\s
```

We don't need match groups, right?